aquarius/app/es_instance.py

```diff
@@ -110,6 +110,14 @@ def read(self, resource_id):
             "_source"
         ]
 
+    def exists(self, resource_id):
+        """Check if document exists.
+        :param resource_id: id of the object to be read.
+        :return: true if object exists
+        """
+        # logger.debug("elasticsearch::read::{}".format(resource_id))
```

Comment:
I think you can remove the comment. I like this approach a lot and I'm pretty sure there's other instances where we use a read + except instead of trying `exists`. For example in some processors. Can you update those instances as well, or create a ticket for this?

Reply:
done it [here](https://github.com/oceanprotocol/aquarius/pull/928/commits/6ef894566d9fbe42dc457a5615d94c4f89355bf7)